client/src/deepscanToken.ts

```diff
@@ -0,0 +1,122 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) S-Core Co., Ltd. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+'use strict';
+
+import * as vscode from 'vscode';
+
+export class DeepscanToken {
+  private readonly _secretStorage: vscode.SecretStorage;
+  private tokenName: string;
+  private serverUrl: string;
+  private tokenRenerateUrl: string;
+
+  constructor(context: vscode.ExtensionContext, serverUrl: string) {
+    this._secretStorage = context.secrets;
+    this._registerCommands();
+    context.subscriptions.push(context.secrets.onDidChange((e) => this._handleSecretChange(e)));
+    this.tokenName = 'deepscan-token';
+    this.serverUrl = serverUrl;
+    this.tokenRenerateUrl = `${serverUrl}/auth/git?redirect_uri=/dashboard/#view=account-settings`;
+  }
+
+  private _validateToken(token: string) {
+   if (!token) {
+      vscode.window.showWarningMessage(`Sorry, DeepScan access token for VS Code cannot be blank.`);
```

Comment:
```suggestion
      vscode.window.showWarningMessage(`Access token cannot be blank.`);
```

"Sorry,"는 뺴주세요.
Sorry는 서버 다운, 알수 없는 서버 오류 등과 같이 우리측의 문제로 기능을 제공해주지 못할 때 쓰는 것이 적절한 것 같습니다.